Scene/WorldSelector/ExtraWorldSelector.tscn

```diff
@@ -1,17 +1,15 @@
-[gd_scene load_steps=3 format=3 uid="uid://b1qffdabwqnxf"]
+[gd_scene load_steps=2 format=3 uid="uid://b1qffdabwqnxf"]
 
 [ext_resource type="Theme" uid="uid://dqt6eyc0l8kms" path="res://Theme/WorldSelector.tres" id="1_kkuy1"]
-[ext_resource type="Script" path="res://Script/WorldSelector/ExtraWorldSelector.gd" id="2_jpepg"]
```

Comment:
This change to detach the script from ExtraWorldSelector is presumably unintentional and breaks this page.